src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--main-text-color);
+
+  background-color: var(--main-ba);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 0 3px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  padding-top: 40px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: var(--secondary-text-color);
+  line-height: 14px;
+
+  padding-top: 4px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 16px;
+}
+
+.card__reviews {
+  font-size: 10px;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 24px;
+}
+
+.card__price {
+  font-size: 10px;
+  color: var(--secondary-text-color);
+  line-height: 18px;
+}
+
+.card__value {
+  font-weight: 700;
```

Comment:
Actually, u also can make var( ) for font-weight with the same values & font-size too:)